src/Microsoft.Data.Sqlite/SqliteFactory.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Data.Common;
+
+namespace Microsoft.Data.Sqlite
+{
+    public class SqliteFactory : DbProviderFactory
```

Comment:
FWIW, many providers use "nickname" + Factory to name this class, e.g. `SqlClientFactory`, `OracleClientFactory` (both the obsolete Microsoft and the Oracle implementation use this), `MySqlClientFactory`, `NpgsqlFactory`. If we care to have shorter handle than Microsoft.Data.Sqlite that people can use in StackOverflow.com and other site, this might be a good way to start :smile: